files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/destinations.yml

```
@@ -1,97 +1,76 @@
 ---
-# ALL tags must be with dashes (-) instead of underscores (_)
+# NOTE: Use dashes (-) exclusively for tags and underscores (_) exclusively for destinations.
+# submit_request_cpus its called in pulsar and in plain condor only request_cpus
 destinations:
-  interactive_pulsar:
-    runner: pulsar_embedded
-    destination_name_override: "embedded_pulsar_interactive_c{cores}-m{mem}"
-    max_accepted_cores: 24
-    max_accepted_mem: 128
-    max_accepted_gpus: 0
+  ######################
+  # BASIC DESTINATIONS #
+  ######################
+  basic_docker_destination:
     params:
-      tmp_dir: true
       docker_enabled: true
       docker_sudo: false
       docker_net: bridge
       docker_auto_rm: true
       docker_set_user: ""
       require_container: true
-      # submit_request_cpus its called in pulsar and in plain condor only request_cpus
       submit_request_cpus: "{cores}"
       submit_request_memory: "{mem}G"
       outputs_to_working_directory: false
       container_monitor_result: callback
-      submit_requirements: 'GalaxyDockerHack == True'
+      submit_requirements: "GalaxyDockerHack == True"
+
+  basic_singularity_destination:
+    params:
+      submit_request_cpus: "{cores}"
+      submit_request_memory: "{mem}G"
+      singularity_enabled: true
+      singularity_volumes: "$_CONDOR_SCRATCH_DIR:rw,$job_directory:rw,$tool_directory:ro,$job_directory/outputs:rw,$working_directory:rw,/data/db/:ro,/data/dnb01/galaxy_db/:ro,/data/dnb02/galaxy_db/:ro,/data/dnb03/galaxy_db/:ro,/data/dnb05/galaxy_db/:ro,/data/dnb06/galaxy_db/:rw,/data/dnb07/galaxy_db/:rw,/data/dnb08/galaxy_db/:rw,/data/dp01/galaxy_db/:rw,/data/0/galaxy_db/:ro,/data/1/galaxy_db/:ro,/data/2/galaxy_db/:ro,/data/3/galaxy_db/:ro,/data/4/galaxy_db/:ro,/data/5/galaxy_import/galaxy_user_data/:ro,/data/6/galaxy_db/:ro,/data/7/galaxy_db/:ro,/usr/local/tools/:ro"
+      singularity_default_container_id: "/cvmfs/singularity.galaxyproject.org/all/centos:8.3.2011"
+
+  ################################
+  # EMBEDDED PULSAR DESTINATIONS #
+  ################################
+
+  interactive_pulsar:
```

```suggestion
  embedded_pulsar:
```
could we use embedded more generic and not only for interactive tools, or is there a reason to limit it?